build.gradle

```diff
@@ -18,9 +18,12 @@ plugins {
     id 'com.github.hierynomus.license' version '0.16.1'
     id 'com.github.spotbugs' version '5.0.5'
     id 'version-catalog'
+    id 'java-library'
```

Comment:
why do we need this java-library plugin here?

Reply:
After testing options I forgot to remove it.
Pushed changes.